src/components/Footer/Footer.tsx

```diff
@@ -1,15 +1,20 @@
 import React from 'react';
 import { Facebook, Instagram, Twitter, GitHub } from '@mui/icons-material';
 import './footer.css';
+import { Link } from 'react-router-dom';
 function Footer() {
   return (
     <div className="footerContainer">
       <div className="topFooter"></div>
       <div className="bottomFooter">
         <div className="miscInfo">
-          <p>About Us</p>
+          <Link to={'./about'}>
```

Comment:
I commented back in the footer just to see how this looks - so it seems that because it's a `Link` now, it gets link stylings, so it looks like this:
<img width="314" alt="Screen Shot 2023-02-15 at 2 48 52 PM" src="https://user-images.githubusercontent.com/71895647/219137038-06afc2ae-7495-4788-9eda-32a03fdf0669.png">

We want them to look like the white ones, right? so we may have to do some stylings on `a` tags to 'undo' how its defaults look.

Reply:
Also - when you click **About Us**, because you've deleted the about component, this now makes the url `http://localhost:3000/about` display the error component?

Reply:
I didn't comment out the footer, I think it was commented out on main and I merged main. I am aware of the color differences since now it has a hyperlink, we can fix it by adding css rule text-decoration: none and color: inherit. once we get the new layout and working footer done.

Reply:
and re about: yes I deleted this one since i have another branch for it and it is in Home folder.

Reply:
ok!